modules/react-native-zoom-free-calling/components/MeetingScheduleModal/index.js

```diff
@@ -0,0 +1,818 @@
+      occurrences: '7',
+      isBy: true,
+      weekly_days: "1",
+      isDayMonthly: true,
+      dayOfMonth: "23",
+      week: "1",
+      day: "1"
+    }
+  });
+  const [errors, setErrors] = useState({
+    topic: '',
+  })
+  const [openStartDate, setOpenStartDate] = useState(false)
+  const [openEndDate, setOpenEndDate] = useState(false)
+  const [openTimezone, setOpenTimezone] = useState(false);
+  const [openOccurrences, setOccurrences] = useState(false);
+  const [openRecurrenceType, setOpenRecurrenceType] = useState(false);
+  const [openRepeatEvery, setOpenRepeatEvery] = useState(false);
+  const [openOccursOn, setOpenOccursOn] = useState(false);
+  const [openWeeks, setOpenWeeks] = useState(false);
+  const [openDays, setOpenDays] = useState(false);
+  const [timezoneList, setTimezoneList] = useState(timezones);
+  const [recurrenceList, setRecurrenceList] = useState([{
+    label: "Daily",
+    value: "1"
+  }, {
+    label: "Weekly",
+    value: "2"
+  }, {
+    label: "Monthly",
+    value: "3"
+  }, {
+    label: "No Fixed Time",
+    value: "-1"
+  }])
+  const [repeatEveryList, setRepeatEveryList] = useState([])
+  const [occursOnList, setOccursOnList] = useState([])
+  const [occurrencesList, setOccurrencesList] = useState([
+    { label: "1", value: "1" },
+    { label: "2", value: "2" },
+    { label: "3", value: "3" },
+    { label: "4", value: "4" },
+    { label: "5", value: "5" },
+    { label: "6", value: "6" },
+    { label: "7", value: "7" },
+    { label: "8", value: "8" },
+    { label: "9", value: "9" },
+    { label: "10", value: "10" },
+    { label: "11", value: "11" },
+    { label: "12", value: "12" },
+    { label: "13", value: "13" },
+    { label: "14", value: "14" },
+    { label: "15", value: "15" },
+    { label: "16", value: "16" },
+    { label: "17", value: "17" },
+    { label: "18", value: "18" },
+    { label: "19", value: "19" },
+    { label: "20", value: "20" }
+  ])
+  const [weeklyList, setWeeklyList] = useState([{
+    label: "Sun",
+    value: "1",
+    isSelected: true
+  }, {
+    label: "Mon",
+    value: "2",
+    isSelected: false
+  }, {
+    label: "Tue",
+    value: "3",
+    isSelected: false
+  }, {
+    label: "Wed",
+    value: "4",
+    isSelected: false
+  }, {
+    label: "Thu",
+    value: "5",
+    isSelected: false
+  }, {
+    label: "Fri",
+    value: "6",
+    isSelected: false
+  }, {
+    label: "Sat",
+    value: "7",
+    isSelected: false
+  }])
+  const [weeksList, setWeeksList] = useState([{
+    label: "First",
+    value: "1",
+    isSelected: true
+  }, {
+    label: "Second",
+    value: "2",
+    isSelected: false
+  }, {
+    label: "Third",
+    value: "3",
+    isSelected: false
+  }, {
+    label: "Fourth",
+    value: "4",
+    isSelected: false
+  }, {
+    label: "Last",
+    value: "-1",
+    isSelected: false
+  }])
+  const [daysList, setDaysList] = useState([{
+    label: "Sunday",
+    value: "1",
+    isSelected: true
```

Comment:
Same logic here, most of this can be extracted into constants, and avoid repetition